.gitignore

```diff
@@ -6,7 +6,11 @@ out/*
 build
 .DS_Store
 */.DS_Store
-lib/glew/glew.pc
+lib/glew/glew.pc*
+client/models/*
```

Comment:
How about if instead of putting the models folder in .gitignore, we committed all of the models? None of them are bigger than 100MB, which is Github's limit for individual files (there's a warning at 50MB, but as far as I can tell it's meaningless)

Reply:
agree, @realmatthewpeng matthew is on it

Reply:
now git pull takes ~a minute lol...